web/res/js/donationForm.js

```diff
@@ -409,6 +409,13 @@ $( function () {
 	store.dispatch( actions.newAddPageAction( 'personalData' ) );
 	store.dispatch( actions.newAddPageAction( 'bankConfirmation' ) );
 
+	// override validity
```

Comment:
just improved the code together with glorious @gbirke which included removing these lines